src/Component.ts

```diff
@@ -3857,6 +3857,11 @@ class Each extends AbstractElementMediator<any[], HTMLElement, Params> {
 	}
 
 	public wire(): void {
+		if (this.getEl().querySelectorAll("template").length === 0) {
```

Comment:
Using querySelectorAll isn't a valid approach, it is could give false positive or false negative matches, as it is a recursive search.